moped-editor/package.json

```diff
@@ -20,6 +21,7 @@
     "aws-amplify-react": "^4.1.22",
     "axios": "^0.19.0",
     "babel-polyfill": "^6.26.0",
+    "client": "0.0.1",
```

Comment:
Can't tell what this library is for:
https://www.npmjs.com/package/client

The code hasn't been updated in 6 years 😸 
https://github.com/ContinuumBridge/client-js

Reply:
Honestly, I don't know what it is either. I installed a couple of packages before deciding on apollo client so maybe it came along for the ride on something. I will delete it on my local copy to confirm its uselessness.